File: 个人网站/forth/js/2.js
$(function(){
	$("#submit").click(function(){
		var username=$("#username").val();
		var password=$("#password").val();
		//发送请求，判断用户名和密码是否正确
		$.post(
				"../java/2",
				{"username":username,"password":password},
				function(data){
					if(data=="登陆成功"){
						//登陆成功之后，跳转页面
						window.location.href="/day18/main.html";
					}else{
						//失败的时候，提供错误的信息
						$("#msg").css("color","red").html(data);
					}
				},
					"text"
				);
	})
})

File: 个人网站/forth/java/2.java
package text.ajax.servlet;

import java.io.IOException;
import java.io.PrintWriter;

import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

@WebServlet("/2")
public class LoginServlet extends HttpServlet{

	protected void service(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
		resp.setContentType("text/html;charset=utf-8");
		PrintWriter writer=resp.getWriter();
		String msg="";
		String username=req.getParameter("username");	
		String password=req.getParameter("password");
		if("YuelinWang".equals(username)&&"189857570wyl.".equals(password)) {
			msg="登陆成功";
		}else {
			msg="用户名或者密码错误";
		}
		writer.write(msg);
	}
}

